bedboss/bedboss.py

```diff
@@ -138,6 +157,15 @@ def run_all(
     output_folder_bedstat = os.path.join(outfolder, "output")
     os.environ["BEDBOSS_OUTPUT_PATH"] = output_folder_bedstat
 
+    if not pm:
```

Comment:
Is this an actual case where a pipeline manager exists already at this point? Is it global? It seems like this will always be not true and need to create `pm` at this point.